tests/cli/test_campaigns.py

```diff
@@ -23,10 +23,11 @@
 
 
 @pytest.mark.asyncio()
-async def test_campaign_cli(uvicorn: UvicornProcess) -> None:
+@pytest.mark.parametrize("api_version", ["v1"])
```

Comment:
ooh, lerning op: what does this pytest idiom do?

Reply:
It sets up a test matrix as a test input. In this case I'm parametrizing the api_version instead of hard-coding it in the calls the tests may make; in future one could add "v2" to the list if such a thing were available and had a similar interface.

You can also use the parametrize mark to set up expected outputs for different inputs in the same matrix, which can be useful, though it can be quite verbose too, e.g., "`v1` should produce X, `v2` should produce Y, etc.)